tests/api/test_graphql.py

```diff
@@ -216,32 +216,36 @@ def test_get_nodes(product_list):
     msg = "There is no node of type {} with pk {}".format(
         nonexistent_item.type, nonexistent_item.pk
     )
-    with pytest.raises(AssertionError, message=msg):
+    with pytest.raises(AssertionError):
```

Comment:
`message` was removed, so I fixed it as they recommended it here -> https://docs.pytest.org/en/latest/deprecations.html#message-parameter-of-pytest-raises

Reply:
Work is being done twice 😕 (#4653)